org.hl7.fhir.utilities/src/main/java/org/hl7/fhir/utilities/cache/PackageHacker.java

```diff
@@ -116,15 +116,36 @@ private void addContentFrom(String folder, Map<String, byte[]> content) throws F
+    if ("file://C:\\GitHub\\hl7.fhir.us.cqfmeasures#2.0.0\\output".equals(webref)) {         return "http://hl7.org/fhir/us/cqfmeasures/STU2"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.davinci-alerts#0.2.0\\output".equals(webref)) {      return "http://hl7.org/fhir/us/davinci-alerts/2020Feb"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.davinci-atr#0.1.0\\output".equals(webref)) {         return "http://hl7.org/fhir/us/davinci-atr/2020Feb";  }
+    if ("file://C:\\GitHub\\hl7.fhir.us.davinci-deqm#1.0.0\\output".equals(webref)) {        return "http://hl7.org/fhir/us/davinci-deqm"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.davinci-deqm#1.1.0\\output".equals(webref)) {        return "http://hl7.org/fhir/us/davinci-deqm/2020Feb"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.davinci-deqm#1.0.0\\output".equals(webref)) {        return "http://hl7.org/fhir/us/davinci-deqm/STU1"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.dme-orders#0.1.1\\output".equals(webref)) {          return "http://hl7.org/fhir/us/dme-orders\\2020May"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.ecr#1.0.0\\output".equals(webref)) {                 return "http://hl7.org/fhir/us/ecr"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.ecr#1.0.0\\output".equals(webref)) {                 return "http://hl7.org/fhir/us/ecr/STU1"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.mcode#1.0.0\\output".equals(webref)) {               return "http://hl7.org/fhir/us/mcode"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.mcode#1.0.0\\output".equals(webref)) {               return "http://hl7.org/fhir/us/mcode/STU1"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.odh#1.0.0\\output".equals(webref)) {                 return "http://hl7.org/fhir/us/odh"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.odh#1.0.0\\output".equals(webref)) {                 return "http://hl7.org/fhir/us/odh/STU1"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.qicore#4.0.0\\output".equals(webref)) {              return "http://hl7.org/fhir/us/qicore"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.qicore#4.0.0\\output".equals(webref)) {              return "http://hl7.org/fhir/us/qicore/STU4"; }
+    if ("file://C:\\GitHub\\hl7.fhir.uv.ips#1.0.0\\output".equals(webref)) {                 return "http://hl7.org/fhir/uv/ips"; }
+    if ("file://C:\\GitHub\\hl7.fhir.uv.ips#1.0.0\\output".equals(webref)) {                 return "http://hl7.org/fhir/uv/ips/STU1"; }
+    if ("file://C:\\GitHub\\hl7.fhir.uv.mhealth-framework#0.1.0\\output".equals(webref)) {   return "http://hl7.org/fhir/uv/mhealth-framework/2020May"; }
+    if ("file://C:\\GitHub\\hl7.fhir.uv.security-label-ds4p#0.1.0\\output".equals(webref)) { return "http://hl7.org/fhir/uv/security-label-ds4p/2020May"; }
+    if ("file://C:\\GitHub\\hl7.fhir.uv.shorthand#0.12.0\\output".equals(webref)) {          return "http://hl7.org/fhir/uv/shorthand/2020May"; }
+       

```

Comment:
Might be better to use a switch statement here.

Reply:
fixed. I'm not used to being able to do switch statements on strings